modules/cms-kit/src/Volo.CmsKit.Public.Web/Pages/CmsKit/Shared/Components/GlobalResources/Style/Default.cshtml

```diff
@@ -0,0 +1 @@
+﻿<link rel="stylesheet" href="/global-resources/style"/>
```

Comment:
`/global-resources/style` is too common. change to `/cms-kit/global-resources/style` .